api/address.proto

```diff
@@ -13,16 +13,25 @@ message Address {
     // matching in FilterChainMatch. For clusters, an address may be either an
     // IP or hostname to be resolved via DNS.
     google.protobuf.StringValue address = 2;
-    oneof port {
+    oneof port_specifier {
```

Comment:
I see. OK. This is fine for now. I will think about it more once it's merged and I can look at it all locally.